### ISSUE TITLE
Question: How to plot training and validation Accuracy?

### ISSUE THREAD

**Issue by kangPrayit** [skorch-dev/skorch#790]
Is there a way how to plot a training and validation accuracy after we finished training with Skorch `net.fit(X_train, y_train)`. 
We can see the `train_loss`, `valid_loss`, and `valid_acc` but how about `train_acc`?
Thank you.

**Comment by BenjaminBossan** [comment]
Yes, that's possible. First, add a [callback](https://skorch.readthedocs.io/en/stable/callbacks.html#skorch.callbacks.EpochScoring) that records the training accuracy:

```python
from skorch.callbacks import EpochScoring
net = NeuralNetClassifier(
    ...
    callbacks=[EpochScoring(scoring='accuracy', on_train=True, name='train_acc')],
)
```

Then, after fitting, retrieve the accuracy scores from the `history`. Here is an example using matplotlib:

```python
plt.plot(net.history[:, 'train_acc'], label='train accuracy')
plt.plot(net.history[:, 'valid_acc'], label='valid accuracy')
plt.legend()
```

**Comment by kangPrayit** [comment]
Thanks, its works like a charm.
